app/grandchallenge/retina_core/templates/retina_base.html

```diff
@@ -9,7 +9,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="">
     <meta name="author" content="">
-
+    <script src="https://browser.sentry-cdn.com/5.6.1/bundle.min.js" integrity="sha384-pGTFmbQfua2KiaV2+ZLlfowPdd5VMT2xU4zCBcuJr7TVQozMO+I1FmPuVHY3u8KB" crossorigin="anonymous"></script>
+    <script>
+        {% if not DEBUG %}
+            // Setup sentry in production before anything else loads
+            Sentry.init({ dsn: 'https://8428444977064be1870761cfc2c1bf35@sentry.io/303639' });
```

Comment:
This shouldn't be hard coded and is available from the environment variable `DJANGO_SENTRY_DSN`. Please:

- Add this as a variable in settings.py
- Get the python `sentry_sdk` to use this variable (line 520)
- Add DJANGO_SENTRY_DSN as a template context variable
- Use that variable in the template